samples/ingestion/ingestion-client/.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "C# (.NET)",
-    "image": "mcr.microsoft.com/devcontainers/dotnet:1-8.0-bookworm",
+    "image": "mcr.microsoft.com/devcontainers/dotnet:1-6.0",
```

Comment:
I saw that all of our .csproj files are configured for .NET 6.0, so I changed the devcontainer to use the same.